edx_proctoring/services.py

```diff
@@ -0,0 +1,42 @@
+"""
+A wrapper class around all methods exposed in api.py
```

Comment:
I really like this pattern -- it would be great if we could establish a similar pattern for LMS integrations -- we sort of have a proto-pattern going with milestones_helpers.py but it would be better to come up with something more like this.